tests/test_circular_imports.py

```diff
@@ -30,14 +31,16 @@ def _mark_aiohttp_worker_for_skipping(
     importables: List[str],
 ) -> List[Union[str, "ParameterSet"]]:
     return [
-        pytest.param(
-            importable,
-            marks=pytest.mark.skipif(
-                not hasattr(socket, "AF_UNIX"), reason="It's a UNIX-only module"
-            ),
+        (
```

Comment:
Which linter wanted this? It seems like too much..

Reply:
Isn't this one just black formatting? I didn't touch this one.